localization/english/dvg_flavor_lotharingia_l_english.yml

```diff
@@ -0,0 +1,101 @@
+
+je_guarani_unite: "Retaking Guarani Lands"
+je_guarani_unite_reason: "Now that we can act as an independent nation, we should refocus on being able to regain our homelands."
+
+je_lot_sol: "Raising Lotharingia's Standard of Living"
+je_lot_sol_reason: "Although the main point of contention is De Aigle's cronyism, civil war progress can be delayed if the standard of living can be raised."
+ #Events
+ 
+reconquering_atlantia.1.t: "The Return of the Golden Province"
+reconquering_atlantia.1.d: "Thanks to our might, we have managed to retake Cote du Cafe."
+reconquering_atlantia.1.f: "A land rich in gold mines and coffee plantations, imagine all the wealth it can bring us! It seems like nothing beats the power of threats and violence if you don't get your way, we would know very well of course."
+reconquering_atlantia.1.a: "Another card for our house!"
+
+dvg_lotharingia.1.t: "The Republic of [ROOT.GetCountry.GetName]"
+dvg_lotharingia.1.d: "After fighting not only for its independence, but for the civil war that ensued right after it, Lotharingia now finds itself under the grip of war hero and despot [ROOT.GetCountry.GetRuler.GetFullName], along with his cronies."
+dvg_lotharingia.1.f: "Despite not being particularly popular with much of the populace, the current ruling class owes its current position in part to their Beornian neighbors, whom they have consistently fought alongside with and mutually recognize. Much like the kingdom to the southwest, many of Lotharingia's elites also intend to keep power within their own families, and would not be opposed to transitioning into a European-style monarchy. However, with the influence of their more liberal neighbors to the east and a declining standard of living in the status quo, maintaining and consolidating power as often been described as a 'House of Cards'. Whatever path Lotharingia hopes to seek however does not seem to be an easy one. Corruption and economic mismanagement is already starting to plague the country and without much reform, another civil war seems inevitable."
+dvg_lotharingia.1.a: "Vive la République!"
+
+dvg_lotharingia.2.t: "Fifty Years of [ROOT.GetCountry.GetName]"
+dvg_lotharingia.2.d: "After consolidating power for so long without any significant opposition or reforms, De Aigle along with his family and friends have been able to enjoy a comfortable rule over the country."
+dvg_lotharingia.2.f: "Without much significant opposition and with De Aigle's health detiriorating, [ROOT.GetCountry.GetName] is at a crossroads. ensuring an even more firm grip over the country, for himself and his family. Odds are if he dies, another De Aigle will probably take his place. The others may not like this, but what are they going to do? Revolt and secede? \n\n Of course, maybe we could just appoint somebody else qualified to take the reins."
+dvg_lotharingia.2.a: "After ruling for so long, it is clear De Aigle has the divine right to rule the nation. Let's establish a new Lotharian Empire!"

```

Comment:
Note: If De Aigle is the current ruler dynasty, you can also use the lastName of your character here.
[ROOT.GetCountry.GetRuler.GetLastName]